zerver/forms.py

```diff
@@ -55,9 +55,10 @@ def email_is_not_mit_mailing_list(email: Text) -> None:
             else:
                 raise AssertionError("Unexpected DNS error")
 
-def check_subdomain_available(subdomain: str) -> None:
+def check_subdomain_available(subdomain: str, from_management_command: bool=False) -> None:
     error_strings = {
         'too short': _("Subdomain needs to have length 3 or greater."),
+        'empty': _("Subdomain cannot be empty."),
```

Comment:
I don't think you need a case for empty, in that empty should be SUBDOMAIN_FOR_ROOT_DOMAIN, right?